QuickStart/inv_NACA0012.cfg

```diff
@@ -261,7 +261,7 @@ CONV_CAUCHY_ELEMS= 100
 % Epsilon to control the series convergence
 CONV_CAUCHY_EPS= 1E-6
 %
-SCREEN_OUTPUT=(INNER_ITER, WALL_TIME, RMS_DENSITY, LIFT, DRAG, CAUCHY_SENS_PRESS, CAUCHY_DRAG RMS_ADJ_DENSITY RMS_ADJ_ENERGY)
+SCREEN_OUTPUT=(INNER_ITER, WALL_TIME, RMS_DENSITY, LIFT@airfoil, DRAG, CAUCHY_SENS_PRESS, CAUCHY_DRAG RMS_ADJ_DENSITY RMS_ADJ_ENERGY)
```

Comment:
That's a cool feature. Would it be possible to specify multiple markers? Something along the lines of `LIFT@(upper_surface, lower_surface)